src/modules/fundingManager/oracle/FM_PC_ExternalPrice_Redeeming_v1.sol

```diff
@@ -494,14 +492,18 @@ contract FM_PC_ExternalPrice_Redeeming_v1 is
             flags: flags,
             data: data
         });
+
+        // Add order to payment client.
         _addPaymentOrder(order);
 
         // Process payments through the payment processor.
         __Module_orchestrator.paymentProcessor().processPayments(
             IERC20PaymentClientBase_v1(address(this))
         );
 
-        // Emit event with all order details.
+        // Emit event with contract address that created the order and with the order ID.
+        emit PaymentOrderCreated(address(this), _orderId);
```

Comment:
This should be included in the redemptionOrderCreated event. I don't think we need 2 events